### ISSUE TITLE
Download crash reports as symbolicated stack traces

### ISSUE THREAD

**Issue by philmitchell** [getsentry/sentry#69176]
### Problem Statement

Sentry currently allows us to download a JSON dump of crash data, including breadcrumbs, tags, etc. That's useful for automation, but not so useful for manually tracking specific crash clusters.

### Solution Brainstorm

Be able to download just the symbolicated stack trace as a text file.

### Product Area

Issues

**Comment by scefali** [comment]
@philmitchell So the ask is you want text instead of json?

**Comment by philmitchell** [comment]
@scefali Yes, but probably a better way to say it is: in human-readable form. Of course one can read the json, but not as readily as a typical stack trace, which is much more compact.

**Comment by lobsterkatie** [comment]
So in other words, you'd like to be able to download this version of the stacktrace?

![image](https://github.com/getsentry/sentry/assets/14812505/f875a2db-f232-4aef-b8d4-1f8be05e3da0)

**Comment by philmitchell** [comment]
Yes. Attached is a (redacted) example of the kind of report I'm used to being able to download for iOS. 
[redacted_crash_report.txt](https://github.com/getsentry/sentry/files/15045760/redacted_crash_report.txt)

**Comment by jangjodi** [comment]
Thank you for the suggestion [philmitchell](https://github.com/philmitchell) - I've added it to our backlog.

**Comment by gaprl** [comment]
Hey @philmitchell, this issue was opened over a year ago. Is this feature still relevant for your workflow? If we don't hear back in the next couple of weeks, we'll close this issue. If you're still interested, it would be helpful to know: Are you still using the JSON export workaround? Would you need this for all platforms or specific ones? Would a simple text format work, or do you need platform-specific formats?

**Comment by philmitchell** [comment]
Hi @gaprl, thanks for following up! In principle, I'd like to have this ability, but it's not a burning issue for me. I don't use the JSON export, currently, I just make do with the web interface. I am only on iOS. Simple text would be great, but if I'm the only one who wants it, I'll understand :)

**Comment by iressler** [comment]
Adding myself as someone who would like this, but also isn't blocked by it not existing.

**Comment by PSinha1202** [comment]
+1 on this. Can we get this prioritized?
